docs/get-started/how-to/poh-api.mdx

```diff
@@ -4,79 +4,60 @@ description: Use the Linea Proof of Humanity API to verify the authenticity of y
 image: /img/socialCards/check-proof-of-humanity.jpg
 ---
 
-import Tabs from "@theme/Tabs";
-import TabItem from "@theme/TabItem";
+Proof of Humanity was originally created to verify community members taking part in the Linea Voyage.
+It aggregated many attestation providers, giving users many options onchain transaction history, biometrics, zero knowledge KYC, and more to demonstrate their personhood, using the Verax Attestation Registry under the hood.
 
-Proof of Humanity was originally created to verify community members taking part in the Linea Voyage. 
-It aggregates many attestation providers, giving users many options (onchain transaction history, 
-biometrics, zero-knowledge KYC, etc.) to demonstrate their personhood, using the [Verax Attestation Registry](https://ver.ax/) 
-under the hood.
+Even though PoH was created for our own needs, the API is publicly available to the community.
+This enables projects to check and verify if an address is owned by a genuine user.
+It can be used to gate access to rewards or special services onchain for example to register a Linea Name.
 
-Even though PoH was originally created for our own needs, the API is publicly available to the 
-community. This enables projects to check and verify if an address is owned by a genuine user. It 
-can be used to gate access to rewards or special services onchain (for example, to register a Linea 
-Name).
+To check and secure PoH status, visit [Sumsub's Proof of Personhood page](https://in.sumsub.com/websdk/p/uni_BKWTkQpZ2EqnGoY7).
 
-To check and secure PoH status, visit the [Linea PoH page](https://poh.linea.build/).
+:::info New in V2
+PoH V2 relies on a single Verax attestation issued by Sumsub instead of multiple attestations issued by multiple actors.
+The endpoint returns a single boolean with no extra payload.
+:::
 
-## Offchain verification
+:::caution V1 deprecated
+The V1 endpoints remain available for a short transition period.
+Deprecation window set to in the next few weeks.
+:::
 
-The API endpoint for the service is: `https://linea-xp-poh-api.linea.build/`
+## Offchain verification V2
+
+The API endpoint for the service is: https://linea-xp-poh-api.linea.build/
```

Comment:
```suggestion
The API endpoint for the service is: `https://linea-xp-poh-api.linea.build/`
```

we need to do this to API endpoints otherwise the link checker interprets them as URLs and tries to access them, causing the checker to fail 😅